src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -298,9 +299,9 @@ export default class AircraftCommander {
      * @param data {array}
      */
     runSpeed(aircraft, data) {
-        const speed = data[0];
+        const nextSpeed = data[0];
 
-        return aircraft.pilot.maintainSpeed(aircraft.speed, speed);
+        return aircraft.pilot.maintainSpeed(aircraft.speed, nextSpeed, aircraft);
```

Comment:
If we're passing the entire `aircraftModel`, there is little point in sending `aircraft.speed` separately. The receivers my method can get that value itself.